mkdocs.yml

```diff
@@ -41,6 +41,7 @@ markdown_extensions:
 nav:
   - index.md
   - requirements.md
+  - releasenotes.md
```

Comment:
@joaocosta-codacy, can you think of an easy solution to:

* Add a new Markdown file per version (`releasenotes-1.5.0.md`, releasenotes-1.6.0.md`, ...). Generating the files is easy, but how easy is it to add new entries to the `nav` block of the `mkdocs.yml` file automatically? :thinking: Perhaps a YAML library can help us do that!
* Opening a new branch / pull request on the [codacy/docs](https://github.com/codacy/docs/tree/master/docs/release-notes/self-hosted) repository to add these new files there instead?

This is so we can have the release notes for all the versions together in the same place, to make it easier for customers to see for example when a new feature was released.

Reply:
I actually do not know but I think that opening a PR on codacy/docs sounds sensible and can be automated :-) What do you think?

Reply:
We can start by opening pull requests on codacy/docs, even if for now they require some manual steps. And later we can automate a bit more. :)

Reply:
This line should also be deleted, since we will not publish the release notes directly on the chart repository.